lib/tasks/db.js

```diff
@@ -297,6 +297,8 @@ function getSequelizeInstance () {
 function getMigrator (type, callback) {
   if (helpers.config.configFileExists() || args.url) {
     var sequelize = getSequelizeInstance();
+    var coffeeRegex = /^(?!skeleton)(.*)\.js$|\.coffee$/;
```

Comment:
initally this was matching all js and coffee files. Now it behaves like this http://rubular.com/r/Dr6tXjsTZ2